### ISSUE TITLE
Why bAcc is always 0

### ISSUE THREAD

**Issue by xinyaojiejie** [IntuitionMachines/OrigamiNet#3]
I have trained the model on icdar datasets. But I observed that bAcc is always 0 even when the iteration is 22000.  Is this normal? Or something wrong? Could you give me sone light on the training of the model?
<img width="1069" alt="5" src="https://user-images.githubusercontent.com/5592259/87412869-1e5c0900-c5fc-11ea-8022-d6a10c43233a.png">

**Comment by ASDen** [comment]
`bAcc` is the best whole sequence accuracy (i.e. for how many sequences/pages in the validation set, did the model get all the characters right without any errors).
So, for complex datasets like ICDAR HTR it is normal to get low `bAcc` even for low `CER`.
From the log, I expect you are using quite low `batch_size` which is slowing down training...

**Comment by xinyaojiejie** [comment]
Thank you for your reply. I will try a large batch_size.

**Comment by sharmaannapurna** [comment]
Hi, I am running the training with batch size=8 on single GPU. The CER seem to be stuck at 0.70 after 22k. I have reached 40K and seems like no improvements. Please let me know if thats how the training continues or am I missing something?
